src/application/components/GitHubReleaseHoverCard.tsx

```diff
@@ -68,27 +69,35 @@ function SnapshotCardContents({ version }: { version: SnapshotVersion }) {
             <div className="flex gap-1 items-center text-xs font-bold uppercase text-secondary dark:text-secondary-dark">
               Published {formatDate(parseSnapshotTimestamp(release.timestamp))}
             </div>
-            <a
+            <ExternalLink
               className="flex gap-1 items-center mt-2"
               href={`https://github.com/apollographql/apollo-client/pull/${release.prNumber}`}
               target="_blank"
               rel="noreferrer"
```

Comment:
```suggestion
```

Can we remove these props once the browser `ExternalLink` is updated to add these props directly?